cmd/kube-scheduler/app/testing/command_test.go

```diff
@@ -0,0 +1,361 @@
+package testing
```

Comment:
Sorry, I was wrong. This should go to `cmd/kube-scheduler/app/server_test.go`. And `Run` doesn't have a test. It's instead being used in some integration tests.

This package is more of a library for other tests.

Reply:
Done